pkg/daemon/daemon.go

```diff
@@ -505,11 +508,11 @@ func (dn *Daemon) setupFileWatcher(watcher *fsnotify.Watcher) error {
 // applySSHTaint calls nodewriter to apply the specified ssh taint
 func (dn *Daemon) applySSHTaint() error {
 	glog.Infof("Detected ssh! The node is being tainted with: %v=%v:%v",
-	MachineConfigDaemonSSHTaintKey, MachineConfigDaemonSSHTaintValue, corev1.TaintEffectNoSchedule)
+		MachineConfigDaemonSSHTaintKey, MachineConfigDaemonSSHTaintValue, corev1.TaintEffectNoSchedule)
```

Comment:
Looks like `gofmt`-like tool in this area too. No biggie, but may be worth dropping as well for now.